services/workers/src/main/resources/dataaccess-worker-spb.xml

```diff
@@ -44,4 +44,42 @@
 		<property name="repeatInterval" value="30000" />
 	</bean>
 	
+	<!-- This worker will process change messages from access approvals in order to send a notification if needed -->
+	<bean id="accessApprovalRevokedNotificationWorker" class="org.sagebionetworks.dataaccess.workers.AccessApprovalRevokedNotificationWorker" scope="singleton" />
+	
+	<bean id="accessApprovalRevokedNotificationTrigger" class="org.springframework.scheduling.quartz.SimpleTriggerFactoryBean"
+		scope="singleton">
+		<property name="jobDetail">
+			<bean
+				class="org.springframework.scheduling.quartz.MethodInvokingJobDetailFactoryBean">
+				<property name="targetObject">
+					<bean
+						class="org.sagebionetworks.asynchronous.workers.changes.ChangeMessageDrivenWorkerStack">
+						<constructor-arg index="0" ref="countingSemaphore" />
+						<constructor-arg index="1" ref="awsSQSClient" />
+						<constructor-arg index="2">
+							<bean
+								class="org.sagebionetworks.asynchronous.workers.changes.ChangeMessageDrivenWorkerStackConfig">
+								<property name="runner" ref="accessApprovalRevokedNotificationWorker" />
+								<property name="semaphoreLockKey" value="accessApprovalRevokedNotificationWorker" />
+								<property name="semaphoreMaxLockCount" value="1" />

```

Comment:
this cannot be increased without adding additional race condition logic to the worker